shared_libraries/orca_shared/database/adapters/api/aws.py

```diff
@@ -1,15 +1,15 @@
 import json
-import logging
 import os
 
 import boto3
+from aws_lambda_powertools import Logger
 
 from orca_shared.database.entities.postgres_connection_info import (
     PostgresConnectionInfo,
 )
 
 
-def get_configuration(db_connect_info_secret_arn: str, logger: logging.Logger) \
+def get_configuration(db_connect_info_secret_arn: str, logger: Logger()) \
```

Comment:
Why does the type-hint have `()`?

Reply:
to initiate the logger. Doing this here directly.
`logger = Logger()`

Reply:
I don't believe it works that way. That's a type-hint, not a default value. Try running the code snippet below, and you'll see that it errors out, as `logger` is still a required parameter.

class TempLogger:
    def __init__(self):
        print("Initializing Logger")


def get_configuration(logger: TempLogger()):
    print(f"In get_configuration. Logger: {logger}")


print("in main")
get_configuration()
print("Wrapping up")

Reply:
changed to 
```
# Set AWS powertools logger
Logger = Logger()


def get_configuration(db_connect_info_secret_arn: str, logger: Logger)
```